### ISSUE TITLE
Server Restart

### ISSUE THREAD

**Issue by LordTheFreak** [EvolSoft/ServerAuth#120]
After the Server Restart, the IP Login wont work, it says you must login ....

**Comment by thelucyclub** [comment]
You should remove the "Possible Bug" label. This isn't that bad, and when the server restarts, you should log back in anyways, just for security. If you really don't like it, I suggest you to not restart your server as much.

**Comment by LordTheFreak** [comment]
Yeah but we have a server Network with over 200 servers and all servers are connected with /transferserver .... if any player joins a game server it says you must login but the player always logging in the lobby server ... ( im From Antoxia.net)

**Comment by LordTheFreak** [comment]
pls add ip check on join

**Comment by LordTheFreak** [comment]
after server restart

**Comment by thelucyclub** [comment]
Ok, then you need accounts stored on a databse, which I'm assuming you don't do. If you do, and IPLogin is true, it should work fine.

**Comment by ghost** [comment]
Not work and database too

**Comment by ghost** [comment]
I mean on database too

**Comment by ghost** [comment]
I mean on database too

**Comment by inxomnyaa** [comment]
You could literally just edit your messages.

**Comment by LordTheFreak** [comment]
Can we work together?


Am 05.08.2017 15:00 schrieb "XenialDan" <notifications@github.com>:

You could literally just edit your messages.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/EvolSoft/ServerAuth/issues/120#issuecomment-320442423>,
or mute the thread
